gensim/models/keyedvectors.py

```diff
@@ -507,11 +508,12 @@ def most_similar(self, positive=None, negative=None, topn=10, restrict_vocab=Non
 
         Returns
         -------
-        list of (str, float)
-            Sequence of (word, similarity).
+        {list of (str, float), numpy.array}
```

Comment:
I can see that by `{A, B}` you mean `either A or B`. Is this a standard form of expression? If not, then perhaps it may be easier to explicitly say `A or B` in order to avoid confusing users. For example, upon seeing this first, I thought the return is some kind of set or dict.

Reply:
Thank you for spotting this. The numpy conventions use this syntax for enumerator types:

```python
order : {'C', 'F', 'A'}
    Description of `order`.
```

However, `or` is the correct operator for a union of types. Fixed in 7d017430.